internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -2,6 +2,8 @@
 // or more contributor license agreements. Licensed under the Elastic License 2.0;
 // you may not use this file except in compliance with the Elastic License 2.0.
 
+// you may not use this file except in compliance with the Elastic License 2.0.
```

Comment:
merge artifact?

Reply:
Ah, good catch, most likely, I didn't add it manually, removing it